go.mod

```diff
@@ -35,7 +35,7 @@ require (
 	github.com/sirupsen/logrus v1.6.0
 	github.com/stretchr/testify v1.7.1
 	github.com/uber-go/tally v3.3.15+incompatible
-	github.com/uber/cadence-idl v0.0.0-20220727202351-6774502e78f4
+	github.com/uber/cadence-idl v0.0.0-20220810110900-c8cf5f3a9720
```

Comment:
Hi @Groxx did you mean this change for cadence-idl?

Reply:
yeah, I suppose this PR was probably waiting on that one being merged, wasn't it.  will read today 👍